lib/openid_connect/client.rb

```diff
@@ -1,6 +1,6 @@
 module OpenIDConnect
   class Client < Rack::OAuth2::Client
-    attr_optional :userinfo_endpoint, :expires_in
+    attr_optional :userinfo_endpoint, :end_session_endpoint, :expires_in
```

Comment:
I don't think you need `end_session_endpoint` and `end_session_uri` in this class.
What is your use-case for them?